grammars/ruby.cson

```diff
@@ -351,7 +351,8 @@
   }
   {
     'match': '''(?x)
-      \\b(
+      (\\b|(?<!\\d[ \\t]|\\d)-)
```

Comment:
In Ruby is the following valid?
```ruby
3
-4
```
Is that equivalent to `3 - 4`?

Previously we've had problems trying to include the negative sign as part of the number.